src/mono/wasm/runtime-test.js

```diff
@@ -5,9 +5,34 @@
 
 //glue code to deal with the differences between chrome, ch, d8, jsc and sm.
 var is_browser = typeof window != "undefined";
+var consoleWebSocket;
+var print;
 
 if (is_browser) {
 	// We expect to be run by tests/runtime/run.js which passes in the arguments using http parameters
+	window.real_print = console.log;
```

Comment:
I want this in asap but after it lands I think we should consider sending all this as json over the websocket and expand it on the xharness side, we should be able to clean up the console output considerably.

pseudo code like
```
JSON.stringify ({ 'method': method, args: {...args}}) 
```
where method is { 'print', 'console.debug', ... }

and after that it might make sense to add a custom log provider so that send even more structure through.